Code/Editor/EditorEngineProcessFramework/PickingRenderPass/PickingRenderPass.cpp

```diff
@@ -113,12 +111,20 @@ void ezPickingRenderPass::Execute(const ezRenderViewContext& renderViewContext,
 
   RenderDataWithCategory(renderViewContext, ezDefaultRenderDataCategories::SimpleOpaque);
 
+
   renderViewContext.m_pRenderContext->SetShaderPermutationVariable("PREPARE_DEPTH", "TRUE");
   RenderDataWithCategory(renderViewContext, ezDefaultRenderDataCategories::SimpleForeground);
 
   renderViewContext.m_pRenderContext->SetShaderPermutationVariable("PREPARE_DEPTH", "FALSE");
   RenderDataWithCategory(renderViewContext, ezDefaultRenderDataCategories::SimpleForeground);
 
+  if (m_bPickTranslucent)
```

Comment:
This works, but not sure if this is entirely correct.

Reply:
Almost, LitTransparent should be rendered after LitMasked and before all the simple stuff to keep render order the same as in the regular pipeline.

---
In reply to: [400517046](https://github.com/ezEngine/ezEngine/pull/180#discussion_r400517046) [](ancestors = 400517046)